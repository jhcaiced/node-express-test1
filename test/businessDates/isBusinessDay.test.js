const config = require('../../src/config')
const request = require('supertest')(`${config.url}/api/v1/businessDates/`)

describe('Is Business Day', () => {
  it('Determine if a given date is a business day', async () => {
    let response = await request.post('isBusinessDay').send({
      date: '2018-11-12'
    }).expect(200)
    expect(response.header['content-type']).toBe('application/json; charset=utf-8')
    expect(response.text).toMatchSnapshot()
    response = await request.post('isBusinessDay').send({
      date: '2018-11-11'
    })
    expect(response.text).toMatchSnapshot()
    response = await request.post('isBusinessDay').send({
      date: '2018-11-13'
    })
    expect(response.text).toMatchSnapshot()
  })
})
