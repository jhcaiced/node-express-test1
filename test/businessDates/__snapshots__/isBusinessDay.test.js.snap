// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is Business Day Determine if a given date is a business day 1`] = `"{\\"ok\\":true,\\"initialQuery\\":{\\"date\\":\\"2018-11-12\\"},\\"results\\":{\\"isBusinessDate\\":false}}"`;

exports[`Is Business Day Determine if a given date is a business day 2`] = `"{\\"ok\\":true,\\"initialQuery\\":{\\"date\\":\\"2018-11-11\\"},\\"results\\":{\\"isBusinessDate\\":false}}"`;

exports[`Is Business Day Determine if a given date is a business day 3`] = `"{\\"ok\\":true,\\"initialQuery\\":{\\"date\\":\\"2018-11-13\\"},\\"results\\":{\\"isBusinessDate\\":true}}"`;
